Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean

```diff
@@ -707,3 +707,41 @@ protected theorem comap (μ : Measure β) [IsFiniteMeasureOnCompacts μ] (f : α
 end MeasureTheory.Measure.IsFiniteMeasureOnCompacts
 
 end BorelSpace
+
+section Support
+
+open MeasureTheory Measure
+
+open scoped Topology
+
+variable {X : Type*} [TopologicalSpace X] [MeasurableSpace X]
+
+variable {μ : Measure X}
+
+lemma support_restrict_subset_closure [OpensMeasurableSpace X] {s : Set X} :
+  (μ.restrict s).support ⊆ closure s := by
+  intro x hx
+  simp only [mem_closure_iff_nhds]
+  rw [(nhds_basis_opens x).forall_iff (fun _ _ h ↦ Set.Nonempty.mono (by gcongr))]
+  intro U ⟨hxU, hU⟩
+  by_cases H : (s ∩ U).Nonempty
+  · exact Set.inter_nonempty_iff_exists_right.mpr H
+  · have h_restr : (μ.restrict s) U = μ (U ∩ s) := by
+      simp [Measure.restrict_apply hU.measurableSet, Set.inter_comm]
+    rw [(nhds_basis_opens x).mem_measureSupport] at hx
+    exact MeasureTheory.nonempty_of_measure_ne_zero
+      (ne_of_gt (h_restr ▸ hx U ⟨hxU, hU⟩))
+
+lemma support_restrict_subset_support {s : Set X} : (μ.restrict s).support ⊆ μ.support := by
```

Comment:
This follows immediately from my suggested lemma `support_mono` (with assumption `μ ≤ ν`), so no need to include it.

Reply:
Fine as is.